system.js

```diff
@@ -17,7 +17,9 @@ var globalEnv = {
   MAIL_PASS: '0383c445ef22d4',
   GOOGLE_API_ENABLED: 'false',
   GOOGLE_MAPS_KEY: 'AIzaSyC3xF9XV91bS2R14Gjmx3UQaKbGgAfHbE4',
-  CDF_ADMINS: 'manager@example.com,admin@example.com'
+  CDF_ADMINS: 'manager@example.com,admin@example.com',
+  BADGEKIT_API_SECRET: 'yoursecret',
```

Comment:
we don't deploy the forum w/ cp-local, does it really makes sense to expose those ?